test/Tests/Readers/LaTeX.hs

```diff
@@ -383,6 +383,12 @@ biblatexCitations :: TestTree
 biblatexCitations = testGroup "biblatex"
   [ "textcite" =: "\\textcite{item1}"
     =?> para (cite [baseCitation] (rt "\\textcite{item1}"))
+  , "textcite with multiple citations" =: "\\textcite{item1,item2}"
+    =?> para (cite [baseCitation] (rt "\\textcite{item1,item2}") <>
+              str "," <>
+              space <>
+              cite [baseCitation {citationId = "item2"}] (rt "\\textcite{item1,item2}")
```

Comment:
I think it should be broken up, so that the first citation gets `\textcite{item1}`, the second `\textcite{item2}`, etc. This might be tricky when there are prefixes, etc.

Reply:
Hi - in the end I went for a different approach, whereby the parsers return the raw text along with the `Citation` (i.e. from `... -> LP m [Citation]` to `... -> LP m [(Citation, Text)]`.

I still need to clean this up, but wanted to check whether you are happy with the approach?